Openwrt feed update

vasvir opened the pull request:
- fix feed to work with LEDE
- Update to apt-cacher-ng-3.1
- Added README.md
- Removed squid file since there is a squid package now in lede

TODO:
- use procd

vasvir committed:
there is a squid package now

vasvir committed:
Update apt-cacher-ng to 3.1 and fix feed

vasvir committed:
README.md updates

vasvir committed:
Update README.md

vasvir committed:
Update README.md

vasvir committed:
Update README.md

vasvir committed:
Update README.md

vasvir committed:
Update README.md

vasvir committed:
Fighting with github's markdown indentation syntax

vasvir committed:
undo: indentation is wrong since we were not on lists

vasvir committed:
Update README.md

vasvir committed:
Update to openwrt 19.07.3 - apt-cacher-ng 3.5

vasvir committed:
fix wrong order in install-time helper blurb

vasvir committed:
typo

vasvir committed:
README markdown fixes

vasvir committed:
remove unecessary Makefile override command

vasvir committed:
undo wrong commit

vasvir committed:
BUGFix: create rundir

vasvir committed:
use the non deprecated PKG_HASH

canaindica commented:
If you plan to update your PR, please consider my PRs (#3 and #4) to use builtin functions to create the user/group and to set the permissions. The two now enable the integration of apt-cacher-ng into a custom image.

vasvir commented:
On 5/14/21 9:07 AM, canaindica wrote:
> If you plan to update your PR, please consider my PRs (#3 <https://github.com/unforgiven512/openwrt-packages/pull/3> and #4 <https://github.com/unforgiven512/openwrt-packages/pull/4>) to use builtin functions to create the user/group and to set the permissions. The two now enable the integration of apt-cacher-ng into a custom image.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/unforgiven512/openwrt-packages/pull/2#issuecomment-841031650>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AA7BCOAPETMJFLWICGOSUE3TNS43FANCNFSM4EJ4ACGQ>.
> 

Do you mind resubmit against https://github.com/vasvir/openwrt-packages I have forked the package because the maintainer was unresponsive and I am not so versed in git to keep a long pull request updated.

Of course if @unforgiven512 is still around I will happily resubmit my patches to him.

    Vassilis

canaindica committed:
Use built-in user add and FILE_MODES routine

Since SVN-Revision 42838 of OpenWrt the creation of a user/group pair during package installation is possible via the USERID option in the package definition section. FILE_MODES can be set too during this process. Thus most of the postinst stuff can be removed. Now the package can be easily integrated into a custom image.

canaindica commented:
Hi,

I resubmited my PRs and concatenated them. By the way, upstream 
apt-cacher-ng version is 3.6.3.

Best regards

Robert

Am 2021-05-14 16:31, schrieb Vassilis Virvilis:
> On 5/14/21 9:07 AM, canaindica wrote:
>> If you plan to update your PR, please consider my PRs (#3
> <https://github.com/unforgiven512/openwrt-packages/pull/3> and #4
> <https://github.com/unforgiven512/openwrt-packages/pull/4>) to use
> builtin functions to create the user/group and to set the permissions.
> The two now enable the integration of apt-cacher-ng into a custom
> image.
>> 
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
> <https://github.com/unforgiven512/openwrt-packages/pull/2#issuecomment-841031650>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7BCOAPETMJFLWICGOSUE3TNS43FANCNFSM4EJ4ACGQ>.
>> 
> 
> Do you mind resubmit against
> https://github.com/vasvir/openwrt-packages I have forked the package
> because the maintainer was unresponsive and I am not so versed in git
> to keep a long pull request updated.
> 
> Of course if @unforgiven512 is still around I will happily resubmit my
> patches to him.
> 
> Vassilis
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [1], or unsubscribe
> [2].
> 
> Links:
> ------
> [1] 
> https://github.com/unforgiven512/openwrt-packages/pull/2#issuecomment-841281299
> [2] 
> https://github.com/notifications/unsubscribe-auth/AOL7ASJNOVLYTCAUZAN2AADTNUX5LANCNFSM4EJ4ACGQ

vasvir committed:
Merge pull request #1 from canaindica/patch-3

Use built-in user add and FILE_MODES routine

vasvir committed:
add chown in chmod in init script instead of the postint section of the Makefile

vasvir committed:
fix up late instructions

vasvir committed:
fix config warnings

vasvir committed:
upgrade to 3.6.3, fix warnings about missing backends by using usptream config files

vasvir committed:
remove custom config dir removal

vasvir committed:
Update README.md
